src/app/shared/components/course-card/course-card.component.ts

```diff
@@ -1,8 +1,27 @@
-import { Component } from '@angular/core';
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { mapAuthorIdsToNames } from '@app/utilities/author-utils';
+import { mockedAuthorsList } from '@shared/mocks/mocks';
```

Comment:
You are reading the mock authors here, but read the mock courses in CoursesListComponent. Can you refactor you code in a way that you read all (mock) data somewhere at the beginning for both sources and pass down the information onto lower level components?

Reply:
I moved it to the parent component(Courses)